src/todolist/settings.py

```
@@ -18,7 +18,7 @@
 # See https://docs.djangoproject.com/en/1.7/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = "@e2(yx)v&tgh3_s=0yja-i!dpebxsz^dg47x)-k&kq_3zf*9e*"
+SECRET_KEY = os.getenv("SECRET_KEY", "")
```

The current code uses `SECRET_KEY = os.getenv("SECRET_KEY", "")`. This reads from the environment (good) but allows an empty string if the env var is missing. To meet the checklist intent and to fail fast when the Secret is not provided, prefer one of these approaches:

- Direct access (KeyError if missing): `SECRET_KEY = os.environ['SECRET_KEY']`

- Or explicit check with clear error:
```python
try:
    SECRET_KEY = os.environ['SECRET_KEY']
except KeyError:
    raise RuntimeError('SECRET_KEY environment variable is required')
```

This prevents the app from starting with an empty SECRET_KEY and makes validation straightforward.